CMakeLists.txt

```diff
@@ -90,6 +92,14 @@ else()
   set(AMBER_ENABLE_DXC FALSE)
 endif()
 
+if (${AMBER_USE_CLSPV})
+  enable_language(ASM)
```

Comment:
What does this do?

Reply:
On my linux GCC build this is required to correctly build LLVM. Otherwise you get errors related to COMPILE_ASM_OBJECT not being defined.